tests/components/lock/test_init.py

```diff
@@ -2,328 +2,261 @@
 from __future__ import annotations
 
 from typing import Any
-from unittest.mock import MagicMock
 
 import pytest
 
 from homeassistant.components.lock import (
     ATTR_CODE,
     CONF_DEFAULT_CODE,
+    DOMAIN,
+    SERVICE_LOCK,
+    SERVICE_OPEN,
+    SERVICE_UNLOCK,
     STATE_JAMMED,
     STATE_LOCKED,
     STATE_LOCKING,
     STATE_UNLOCKED,
     STATE_UNLOCKING,
-    LockEntity,
     LockEntityFeature,
 )
 from homeassistant.core import HomeAssistant
 import homeassistant.helpers.entity_registry as er
-from homeassistant.setup import async_setup_component
+from homeassistant.helpers.typing import UNDEFINED, UndefinedType
 
 from tests.testing_config.custom_components.test.lock import MockLock
 
+TEST_ENTITY_ID = "lock.test_lock"
```

Comment:
Maybe be even more explicit ?
```suggestion
TEST_LOCK_ID = "lock.test_lock"
```

Reply:
Renamed to `TEST_LOCK_ENTITY_ID`